fs/nfs/Kconfig

```diff
@@ -13,6 +13,15 @@ config NFS
 
 #if NFS
 
+config NFS_DONT_BIND_TCP_SOCKET
+	bool "Do not bind TCP socket"
+	default n
+	depends on NFS
```

Comment:
Do we need

```suggestion
	depends on TCP
```
?

Reply:
@pkarashchenko 
If I applied your suggestion, the indentation would be incorrect.

<img width="283" alt="image" src="https://github.com/apache/nuttx/assets/25843920/22587e4c-7d2a-48ad-aec2-59adea619103">

So let me keep the config.

Reply:
Very strange as if I add such a condition all looks fine on my end. Let me submit a PR, so you can try it out

Reply:
@masayuki2009 could you please check https://github.com/apache/nuttx/pull/9668?